index.js

```diff
@@ -124,6 +132,10 @@ var render = module.exports = function(dom, opts) {
 };
 
 function renderTag(elem, opts) {
+  // Handle SVG
+  var oldXmlMode = opts.xmlMode;
+  if (elem.name == "svg") opts.xmlMode = true;
```

Comment:
This is actually a pretty good idea, I would prefer creating a copy of the options object, though.